examples/cryptocurrency-advanced/frontend/src/plugins/blockchain.js

```diff
@@ -4,200 +4,142 @@ import * as proto from '../../proto/stubs.js'
 
 const TRANSACTION_URL = '/api/explorer/v1/transactions'
 const PER_PAGE = 10
-const SERVICE_ID = 128
+const SERVICE_ID = 3
 const TX_TRANSFER_ID = 0
 const TX_ISSUE_ID = 1
 const TX_WALLET_ID = 2
-const TABLE_INDEX = 0
 const Wallet = Exonum.newType(proto.exonum.examples.cryptocurrency_advanced.Wallet)
 
-function TransferTransaction(publicKey) {
-  return Exonum.newTransaction({
-    author: publicKey,
-    service_id: SERVICE_ID,
-    message_id: TX_TRANSFER_ID,
+function TransferTransaction () {
```

Comment:
The transaction doesn't need to be created every time from scratch. Instead, you can define the type once in the module scope:

```javascript
const TransferTransaction = new Exonum.Transaction({
  serviceId: SERVICE_ID,
  methodId: TX_TRANSFER_ID,
  schema: proto.exonum.examples.cryptocurrency_advanced.Transfer
})
```